SU-HSDO/stanford_mrc
# MRCD8-237 address bouncing menu on refresh

# READY FOR REVIEW

# Summary
- Fixes the jumpy-ness that was occurring on navigation clicks and page reloads in the main menu
- The fallback styles and styles for when there is no JS needed to be cleaned up and made more and less specific in places

# Needed By (Date)
- End of sprint (4/11)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the main menu doesn't jump upon reloads or nav clicks

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-327

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
